lib/spack/spack/relocate.py

```diff
@@ -86,6 +98,31 @@ def get_relative_rpaths(path_name, orig_dir, orig_rpaths):
     return rel_rpaths
 
 
+def set_placeholder(dirname):
+    """
+    return string of @'s with same length
+    """
+    return '@' * len(dirname)
+
+
+def get_placeholder_rpaths(path_name, orig_rpaths):
+    """
+    Replaces orig_dir with relative path from dirname(path_name) if an rpath
+    in orig_rpaths contains orig_path. Prefixes $ORIGIN
```

Comment:
Does this docstring still need to be updated? Not sure what `ORIGIN` is referring to here.